src/layouts/SiteLaunchPad/components/SiteLaunchCheckList.tsx

```diff
@@ -385,14 +381,12 @@ export const SiteLaunchChecklistBody = ({
         </TableContainer>
       </Box>
 
-      <Box display="flex" justifyContent="flex-end">
-        <Button
-          variant="link"
-          ml="auto"
-          onClick={() => setPageNumber(SITE_LAUNCH_PAGES.DISCLAIMER)}
-        >
```

Comment:
Do we not need the disclaimer anymore?

Reply:
oh hey this is not a disclaimer, but a back page behaviour. Once you have agreed to the disclaimer, user should not be able to go back to disagree with the disclaimer anymore